preprocessors/azure.py

```
@@ -25,53 +27,61 @@ def __call__(self):
+                concept_config.data_path = join(self.cfg.register_concepts.dump_path, concept_config.filename)
+                concept_config.data_store = self.cfg.register_concepts.data_store
+                if isinstance(concept_config.filename, list):
+                    for file_name in concept_config.filename:
+                        concept_config.filename = file_name
+                        self.iterate_through_file(concept_type, concept_config, first=first, kwargs={'forl': forl, 'kont': kont, 'mapping': mapping})
+                        first=False
+                else:
+                    self.iterate_through_file(concept_type, concept_config, kwargs={'forl': forl, 'kont': kont, 'mapping': mapping})
+        self.save(self.adm_file, self.cfg.admissions, 'admissions')
+
+    def iterate_through_file(self, concept_type, concept_config, first=True, kwargs={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)